docs/product/metrics/metrics-explorer-page.mdx

```diff
@@ -10,27 +10,53 @@ Once you've started sending metrics to Sentry, you can explore them in the produ
 
 ## Querying & Visualization
 
-![Metrics Query](./img/metrics-explorer-query.png)
-
-You can use the dropdowns at the top of the page as a query editor for your metrics. Select the metric you want to plot from a list of available metrics in your project, the aggregate operation you'd like to perform (for example avg, min, max, percentiles, and so on), and, optionally, any tags you'd like to group by. In the image above, the p95 values of the metric, `gibpotato.potatoes.event_processing_time`, is queried and shown, grouped by `event_type`.
-
-Optionally, you can also use the search bar to filter for specific tag values. In addition to custom metrics that are defined by your team, you can also explore any performance metrics that Sentry offers out-of-the-box.
+<div style={{"height":"0px","paddingBottom":"calc(56.8359% + 41px)","position":"relative","width":"100%"}}>
```

Comment:
Small nit, but I noticed that the arcade hint points to the p99 value in the aggregation drop-down, but the instruction says "let's look at the p95 values" and similarly to "release" when it's talking about "event type"

Reply:
I see now that this is a general issue with arcades, but only in Safari, not in Chrome. The pointer is somehow shifted 🙈

Reply:
Oh that's interesting! I had no idea the pointer shifts in other browsers. Thanks for the thorough review!